static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
+    {`There was an error (Error ${code}: ${message}). Please contact `}
+    <a href="mailto:micromasters-support@mit.edu">
```

Comment:
This is actually defined in the django settings as `EMAIL_SUPPORT`. Can you add it to `SETTINGS` and swap it in here?